library/src/main/java/com/bumptech/glide/load/engine/bitmap_recycle/SizeStrategy.java

```diff
@@ -111,7 +111,7 @@ static String getBitmapString(int size) {
   static class KeyPool extends BaseKeyPool<Key> {
 
     public Key get(int size) {
-      Key result = get();
+      Key result = super.get();
```

Comment:
Out of curiosity, what does the super do?

Reply:
Open the file on `master`, there's a warning on it. It clarifies which `get` is being called: both `BaseKeyPool` and `SizeStrategy` have a `get` name in scope, so the super clears it up which one. Actually let me check that again, because it's a static inner class.

Reply:
![image](https://user-images.githubusercontent.com/2906988/34450685-2b87015e-ed08-11e7-9037-a9f36464dac3.png)
so the circumstances I described above stand, but it may be an inspection false positive as the inner class is `static` and the `get` signature is different in the outer class. I guess it still plays nicer with the `super.`.

Reply:
https://youtrack.jetbrains.com/issue/IDEA-184421